src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -38,6 +37,8 @@ object VertexSchema {
   val allClasses = Seq(Workspace, Entity, MethodConfig, MethodRepoConfig, MethodRepoMethod, Submission, Workflow, WorkflowFailure, Map)
 
   def vertexClassOf[T: TypeTag]: String = typeOf[T].typeSymbol.name.decodedName.toString
+  def vertexClassOf(tpe: Type): String  = tpe.typeSymbol.name.decodedName.toString
+  //def vertexClassToType(str: String): Type = ru.runtimeMirror(getClass.getClassLoader).staticClass(str.replace("^", ".")).selfType
```

Comment:
This is unused but I want to leave it here for a mysterious future where it might be needed.